distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -867,6 +874,95 @@ public String getOption() {
             .map(Path::toAbsolutePath)
             .map(Path::toString)
             .map(s -> StringUtils.replace(super.getOption(), "{karaf.home}", s))
+            .map(s -> s.replace('\\', '/'))
+            .map(s -> s.replace("/bin/..", "/"))
+            .orElseGet(super::getOption);
+      } catch (IOException e) {
+        throw new RuntimeException("Unable to determine current exam directory", e);
```

Comment:
❓ What about using AssertionError instead?

Reply:
This is all test code. I don't care what exception is used as long as you're able to see the error.